CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Removed faulty `within_the_last()` method from `sdk.queries.alerts.filters.alert_filter.DateObserved`.
 
+### Changed
+- Updated Alert `resolve-alert` and `reopen-alert` endpoints to `update-state` endpoint.
+
+### Added
+
+- `sdk.alerts.update_state()` method to update state, `sdk.alerts.resolve` and `sdk.alerts.reopen` are deprecated and will be removed from version 2.0
```

Comment:
I'd like to know others thoughts on removing `resolve()` and `reopen()`... Maybe it is too soon to mark these as deprecated

Reply:
I think I'm ok with removing them in 2.0 where breaking changes are expected. Is your concern that it's not enough time for deprecating? Or that you prefer having the explicit methods vs the more generic one?

Reply:
My only hesitation is that `resolve()` and `reopen()` seem (keyword: _seem_) like they would be expected methods to have on an Alert... What can you do with alerts?  You can resolve them.  You can reopen them.... The other states do not so easily come to mind.

Reply:
That being said, I am still ok with deprecating them just because `change_state(AlertState.RESOLVED)` seems perfectly readable to me...

Reply:
Definitely don't remove these now. If we arent sure about whether or not these should be deprecated, they probably shouldn't be. I know that there are multiple statuses for alerts now, but opening/closing will still probably be the most common actions against them, and a convenience method for this seems like a good idea, at least to me. We don't need to remove methods just because the API does.

Reply:
@alanag13 We are not talking about removing them, we are talking about planning to remove them in 2.0, but I agree that this decision can more easily made post this PR so @kiran-chaudhary You might as well remove the deprecation notes.